static/locales/fr/translation.json

```diff
@@ -131,15 +131,15 @@
 		"confirm": "Valider",
 		"cancel": "Annuler",
 		"close": "Fermer",
-		"exit": "Quittez les Paramètres pour Appliquer les Changements",
-		"public_lobby": "Lobbies Publiques",
+		"exit": "Quittez les pParamètres pour appliquer les changements",
```

Comment:
Typo plus we should probably stick with capitalization that the English version has